### DISCUSSION TITLE
How to see nushell error trace?

### DISCUSSION THREAD

**Discussion by markfswe** [nushell/nushell#11323] [Q&A]
I have set nushell as a login shell, after login attempt I get an "permission denied" error. I wonder what does nushell tries to do in order to get this error, is it possible to see what comes before the error? Because currently the only output I get is
```
Error: nu::shell::io_error
× I/O error
help: Os { code: 13, kind: PermissionDenied, message: "Permission denied" }
```

Related: https://github.com/nushell/nushell/issues/7414
And maybe its https://github.com/nushell/nushell/issues/11142 for me

**Comment by amtoine** [comment]
i don't think it's possible to have more information once you get in that state :thinking:

**Reply by markfswe** [reply]
oh.. okay :(
